client/termsetting/handlers/geneExpression.ts

```diff
@@ -29,7 +29,7 @@ export async function fillTW(tw: GeneExpressionTW, vocabApi: VocabApi, defaultQ:
 	if (typeof tw.term.gene != 'string' || !tw.term.gene) throw 'geneExpression tw.term.gene must be non-empty string'
 	if (!tw.term.name) tw.term.name = tw.term.gene // auto fill if .name is missing
 
-	if (!tw.q?.mode) tw.q = { mode: 'continuous' } // supply default q if missing
```

Comment:
any reason to default to discrete?

Reply:
See in the PR description

Reply:
default q=discrete should be set in controls.divide.js rather than here. so to maintain behavior of showing violin for gene exp. i will change it